### PR TITLE
Add suport for HTTP2 assets (cogs-client v2.1)

### PR DESCRIPTION
**Pull request by chetbox**

### CODE REVIEW

File: src/components/Images.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default function Images({
     <img
       className={className}
       key={index}
-      src={assetUrl(image.file)}
+      src={connection.getAssetUrl?.(image.file) ?? assetUrl(image.file)}
```
Comment (matt-oakes) [inline src/components/Images.tsx:29]: Why would this be needed? Wouldn't you always have v2.1.0 of the client lib so this method will always exist?

**Review by matt-oakes — COMMENTED** [review]
Looks good! Just a quick question

File: src/components/Images.tsx (hunk 1 above)
Comment (chetbox) [reply src/components/Images.tsx:29]: I think it's possible for your package-lock.json or yarn.lock to point to `cogs-client@2.0.0` with `cogs-client-react@2.1.0`.

The alternative would be to set the `cogs-client` dependency in this repo to be `>=2.1.0`. What do you think?

Comment (chetbox) [reply src/components/Images.tsx:29]: I think `cogs-client@~2.1` would also work?

Comment (chetbox) [reply src/components/Images.tsx]: I've pushed an update that uses `cogs-client@~2.1` and removes this fallback logic.

Comment (matt-oakes) [reply src/components/Images.tsx]: I think that's correct 👍

**Review by matt-oakes — APPROVED** [review]

**Pull request merged**